examples/webgl_animation_multiple.html

```diff
@@ -1,390 +1,335 @@
-			//////////////////////////////
-
-
-			//////////////////////////////
-			// Information about our 3D models and units
-			//////////////////////////////
-
-			// The names of the 3D models to load. One-per file.
-			// A model may have multiple SkinnedMesh objects as well as several rigs (armatures). Units will define which
-			// meshes, armatures and animations to use. We will load the whole scene for each object and clone it for each unit.
-			// Models are from https://www.mixamo.com/
-			const MODELS = [
-				{ name: "Soldier" },
-				{ name: "Parrot" },
-			];
-
-			// Here we define instances of the models that we want to place in the scene, their position, scale and the animations
-			// that must be played.
-			const UNITS = [
-				{
-					modelName: "Soldier", // Will use the 3D model from file models/gltf/Soldier.glb
-					meshName: "vanguard_Mesh", // Name of the main mesh to animate
-					position: { x: 0, y: 0, z: 0 }, // Where to put the unit in the scene
-					scale: 1, // Scaling of the unit. 1.0 means: use original size, 0.1 means "10 times smaller", etc.
-					animationName: "Idle" // Name of animation to run
-				},
-				{
-					modelName: "Soldier",
-					meshName: "vanguard_Mesh",
-					position: { x: 3, y: 0, z: 0 },
-					scale: 2,
-					animationName: "Walk"
-				},
-				{
-					modelName: "Soldier",
-					meshName: "vanguard_Mesh",
-					position: { x: 1, y: 0, z: 0 },
-					scale: 1,
-					animationName: "Run"
-				},
-				{
-					modelName: "Parrot",
-					meshName: "mesh_0",
-					position: { x: - 4, y: 0, z: 0 },
-					rotation: { x: 0, y: Math.PI, z: 0 },
-					scale: 0.01,
-					animationName: "parrot_A_"
-				},
-				{
-					modelName: "Parrot",
-					meshName: "mesh_0",
-					position: { x: - 2, y: 0, z: 0 },
-					rotation: { x: 0, y: Math.PI / 2, z: 0 },
-					scale: 0.02,
-					animationName: null
-				},
-			];
-
-			//////////////////////////////
-			// The main setup happens here
-			//////////////////////////////
-			let numLoadedModels = 0;
-			initScene();
-			initRenderer();
-			loadModels();
-			animate();
-			//////////////////////////////
-
-
-			//////////////////////////////
-			// Function implementations
-			//////////////////////////////
-			/**
-			 * Function that starts loading process for the next model in the queue. The loading process is
-			 * asynchronous: it happens "in the background". Therefore we don't load all the models at once. We load one,
-			 * wait until it is done, then load the next one. When all models are loaded, we call loadUnits().
-			 */
-			function loadModels() {
-
-				for ( let i = 0; i < MODELS.length; ++ i ) {
-
-					const m = MODELS[ i ];
-
-					loadGltfModel( m, function () {
-
-						++ numLoadedModels;
-
-						if ( numLoadedModels === MODELS.length ) {
-
-							console.log( "All models loaded, time to instantiate units..." );
-							instantiateUnits();
-
-						}
-
-					} );
-
-				}
 
+<head>
```

Comment:
I don't think this formatting is right. `<head>` as well as `<body>` are childs of the `<html>` tag. They should not be positioned on the same level.

Reply:
Ah ok, yes never thought about that. Not on purpose yet most html formatters make a special case for the head and the body without indentation.